src/app/containers/Recommendations/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { string } from 'prop-types';
+import 'isomorphic-fetch';
+import webLogger from '#lib/logger.web';
+
+const logger = webLogger();
+
+const RecommendationsContainer = ({ path }) => {
```

Comment:
Maybe this should be named `CpsRecommendations`? As I don't think this is expected to be used on other pages. Although I'm not sure, because Includes and Social Embeds have been done in this way but the rest of the OJ's use the `Cps` prefix 🤷‍♂️.